.github/workflows/ci.yml

```
@@ -2,15 +2,26 @@ name: CI
 
 on:
```

Honestly probably worth merging these entirely:

```suggestion
on: [push, pull_request]
```

---

Merging these would trigger the CI on every change of every file on every branch and pull request. This is not the purpose of the PR. Not every commit needs to trigger the CI (e.g a README change)

---

I don't follow? The path filter would still apply, no?

---

[It's best to follow Github's example and appropriate YAML syntax.](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths)

---

…I'm not sure what you're pointing at?

---

The path filter would not apply with the suggested changes. What I linked is a working example, which mirrors the changes I've introduced which would not work with the suggested changes since it's invalid YAML.